gadgets/trace_open/test/trace_open_test.go

```diff
@@ -46,14 +46,18 @@ type traceOpenEvent struct {
 func TestTraceOpen(t *testing.T) {
 	gadgettesting.RequireEnvironmentVariables(t)
 
-	cn := "test-trace-open"
 	containerFactory, err := containers.NewContainerFactory("docker")
 	require.NoError(t, err, "new container factory")
 
+	cn := "test-trace-open"
```

Comment:
what about using containerName? I always think of Cartoon Network when reading cn...